src/mame/drivers/nes_vt.cpp

```diff
@@ -99,6 +99,11 @@ class nes_vt_state : public nes_base_state
 	DECLARE_READ8_MEMBER(spr_r);
 	DECLARE_READ8_MEMBER(chr_r);
 
+	/* expansion nametable */
+	std::vector<uint8_t> m_ntram;
```

Comment:
I'm trying to keep it as close to the NES code as possible without giving it a full NES bus etc. which it doesn't have.